packages/core/src/form-field/FormHelperText.css

```diff
@@ -39,11 +39,15 @@
   font-style: var(--uitkFormFieldHelperText-fontStyle, var(--uitk-text-help-fontStyle));
   letter-spacing: var(--uitkFormFieldHelperText-letterSpacing, 0);
   line-height: var(--uitkFormFieldHelperText-lineHeight, var(--uitk-text-base-lineHeight));
-  min-height: var(--uitkFormFieldHelperText-minHeight, var(--uitk-text-caption-minHeight));
+  min-height: var(--uitkFormFieldHelperText-minHeight, var(--uitk-text-help-minHeight));
   margin-bottom: var(--uitkFormFieldHelperText-marginBottom, 0);
   margin-top: var(--uitkFormFieldHelperText-marginTop, var(--formField-withHelperText-marginTop));
```

Comment:
typo `--formField-withHelperText-marginTop`

Reply:
this isn't a typo but it is confusing, essentially the withHelperText token was used as the final version consuming either -helperText-marginTop (default) or -helperText-marginTop-fullWidth
i've changed it now to be -helperText-marginTop consuming either -helperText-marginTop-default or -helperText-marginTop-fullWidth